test/gerber-parser_drill_test.js

```diff
@@ -379,6 +379,28 @@ describe('gerber parser with gerber files', function() {
 
       p.write('X1Y1\n')
     })
+
+    it('should warn / detect trailing zeros if possible', function(done) {
+      var warnings = []
+
+      p.on('warning', function(w) {
+        warnings.push(w)
+      })
+
+      p.on('finish', function() {
+        expect(warnings[0].message).to.match(/assuming trailing/)
+        expect(warnings[1].message).to.match(/detected leading/)
+        expect(p.format.zero).to.equal('L')
+        done()
+      })
+
+      p.write(';FILE_FORMAT=2:4\n')
```

Comment:
Oh I see! Carry on then. Could use expect.satisfy... but it gets pretty verbose. Maybe it's time to switch this codebase over to ES6ish to get some arrow functions?

``` js
expect(warnings).to.satisfy(function (warns) {
  return warns.some(function (w) {
    return /detected leading/.test(w.message)
  })
})
```